.golangci.yaml

```diff
@@ -1,6 +1,9 @@
 run:
   timeout: 5m
 
+issues:
+  max-same-issues: 0 # don't hide issues in CI runs because they are the same type
```

Comment:
This is a small quality-of-life improvement, I hope: Previously, if the linter found a total of 5 issues, but they all had the same type, it wouldn't show you all of them. Now, we ask it to just spit it all out, all at once.